Add back the internal database configuration docs (#1689)
ManageIQ/manageiq-documentation: _includes/external-configuration-db.md

specific settings for correct operation. For example, it must correctly
reclaim table space, control session timeouts, and format the PostgreSQL
server log for improved system support. It is recommended that external databases use a
`postgresql.conf` file based on the standard file used by the
{{ site.data.product.title_short }} appliance.

Ensure you configure the settings in the `postgresql.conf` to suit your
system. For example, customize the `shared_buffers` setting according to
the amount of real storage available in the external system hosting the
PostgreSQL instance. In addition, depending on the aggregate number of
appliances expected to connect to the PostgreSQL instance, it may be
necessary to alter the `max_connections` setting.

**Note:**

  - {{ site.data.product.title_short }} requires PostgreSQL version 10.

  - Because the `postgresql.conf` file controls the operation of all
    databases managed by a single instance of PostgreSQL, do not mix
    {{ site.data.product.title_short }} databases with other types of databases in a single
    PostgreSQL instance.

1.  Start the appliance and open a terminal console.

> 2.  Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance
>     summary screen displays.

Reviewer: ```suggestion
2.  Enter the `appliance_console` command. The summary screen will be displayed.
```
